### ISSUE TITLE
What Do I do about this, seems important?

### ISSUE THREAD

**Issue by Vikng412** [Shabinder/SpotiFlyer#1165]
I think this may be important.
![image](https://user-images.githubusercontent.com/78209058/156571209-3c7b0262-9ad6-4a96-8083-ed0062d007b2.png)

**Comment by Shabinder** [comment]
Yes, the app wont work as expected until you install FFmpeg in your device at PATH.
do a google search for above and you will find plenty of resources to fix this.
